apps/web/app/(authenticated)/(app)/app/banner.tsx

```diff
@@ -9,18 +8,9 @@ import Link from "next/link";
 /**
  * Shows a banner if necessary
  */
-export const UsageBanner: React.FC = async () => {
-  const tenantId = getTenantId();
-
-  const workspace = await db.query.workspaces.findFirst({
-    where: (table, { and, eq, isNull }) =>
-      and(eq(table.tenantId, tenantId), isNull(table.deletedAt)),
-    with: {
-      apis: {
-        where: (table, { isNull }) => isNull(table.deletedAt),
-      },
-    },
-  });
```

Comment:
This part is not necessary as the the parent already cast that query. Better to pass it as prop.

Reply:
it would be fine as that's cached and doesn't impact the page negatively